examples/examples/balance_transfer.rs

```diff
@@ -44,13 +44,30 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         .build()
         .await?
         .to_runtime_api::<polkadot::RuntimeApi<DefaultConfig, PolkadotExtrinsicParams<DefaultConfig>>>();
+
+    // Validate full metadata compatibility.
+    api.validate_metadata()?;
+
+    // Skip the pallet metadata compatibility check.
     let hash = api
         .tx()
-        .balances()
+        .balances_unchecked()
         .transfer(dest, 123_456_789_012_345)
         .sign_and_submit_default(&signer)
         .await?;
 
+    println!(
+        "Balance transfer extrinsic submitted (without pallet metadata validation): {}",
+        hash
+    );
+
+    // Validate pallet metadata compatibility.
```

Comment:
I reckon this comment might be better right above or next to the `.balances()?` bit, since most of this code is just about transferring tokens, and the comment is only really about the one line. Same with the `// Skip the pallet metadata..` comment above

Reply:
Have decided to move to a different example, that we can further extend. Have incorporated the comment style in the `metadata_compatibility` :D